eiffel-vocabulary/EiffelMetaProperty.md

```diff
@@ -0,0 +1,150 @@
+<!---
```

Comment:
I think we should update https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/the-meta-object.md and instead link to that one from each event.